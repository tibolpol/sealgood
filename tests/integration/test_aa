#!/bin/bash

export mydir="$(cd "$(dirname "$0")";pwd)" myname="$(basename "$0")"
. "$mydir/test_prereq"
exec 1> >(tee $mydir/$myname.stdout)


####################################################################
# lookup/PEM pubkey                                                #
# date/PEM pubkey                                                  #
# get_payload                                                      #
# locale/{{run,data}/fr,{run,data}/es,{run,data}/pt,{run,data}/us} #
####################################################################
parallel "mydif $LINENO ../samples/test_date_{} < <((
LANGUAGE={} sealgood clean date < $mydir/../samples/test_date_pt 2>/dev/null) 2>&1 | sed -e 's/\(^MIIV\).*/\1/' )
" ::: us es fr pt
