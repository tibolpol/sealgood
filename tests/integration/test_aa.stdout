ok   15 ../samples/test_date_fr
ok   15 ../samples/test_date_us
ok   15 ../samples/test_date_es
ok   15 ../samples/test_date_pt
